### DISCUSSION TITLE
Documentation related to `getQueryState` seems wrong

### DISCUSSION THREAD

**Discussion by lgenzelis** [TanStack/query#3023] [Q&A] [answered]
**Describe the bug**
[The docs say that getQueryState](https://react-query.tanstack.com/reference/QueryClient#queryclientgetquerystate) accepts a `filters?: QueryFilters` parameter. Is that right? It doesn't make much sense to me. What would `getQueryState` return if we pass e.g. `{exact: false}`? An array of states matching the query? I don't think so (specially since the function is written as `getQueryState` instead of using the plural "queries" like most other functions 😅).

**Comment by TkDodo** [comment, marked as answer]
it currently accepts a `key` and optionally filters. you can set `exact` to `false` to get the first query returned that matches that key. I don't think it is recommend for `getQueryState`, but you can do it. The `filters` are just kept in the same shape everywhere, and there are methods where it makes sense, and it also defaults to `true` for methods that have the plural word in their name, like `invalidateQueries`

for some context, see: https://github.com/tannerlinsley/react-query/issues/1725

also, moving this to discussion because it's not a bug.

**Reply by lgenzelis** [reply]
Got it, thanks @TkDodo ! I opened an issue because I was 100% sure it was a mistake in the docs 😅 . 
 When you say 
>  defaults to true for methods that have the plural word in their name, like invalidateQueries

I guess you mean `defaults to false`.
